src/data_snack/connections/memcached.py

```diff
@@ -11,15 +11,23 @@ class MemcachedConnection(Connection):
     def get(self, key: Text) -> bytes:
         return self.connection.get(key)
 
-    def set(self, key: Text, value: Text) -> bool:
-        return self.connection.set(key, value)
+    def set(self, key: Text, value: Text, expire: int = 0) -> bool:
+        return self.connection.set(key, value, expire=expire)
+
+    def delete(self, key: Text) -> bool:
+        return self.connection.delete(key, noreply=False)
 
     def get_many(self, keys: List[Text]) -> Dict[Text, bytes]:
         return self.connection.get_many(keys)
 
-    def set_many(self, values: Dict[Text, Text]) -> List[Text]:
-        failed_keys = self.connection.set_many(values)
+    def set_many(self, values: Dict[Text, Text], expire: int = 0) -> List[Text]:
+        failed_keys = self.connection.set_many(values, expire=expire)
         return list(set(values.keys()) - set(failed_keys))
 
+    def delete_many(
+        self, keys: List[Text]
+    ) -> bool:  # always returns True. How should we approach this?
```

Comment:
Hmm I dont see such a statement here: "True. If an exception is raised then all, some or none of the keys may have been deleted. Otherwise all the keys have been sent to memcache for deletion and if noreply is False, they have been acknowledged by memcache.". It always returns true then